package.json

```diff
@@ -91,6 +91,6 @@
     "dlv": "^1.1.2"
   },
   "peerDependencies": {
-    "preact": "*"
+    "preact": ">=10 || ^10.0.0-alpha.0"
```

Comment:
We still want to support as much of the preact version history as possible, so this _should_ either continue to be `*` if we can support every version, or at a minimum something like `>=6` or some minimally tested version that we know it works with.

- [ ] Fix peer dependency version range